README.md

```diff
@@ -1,3 +1,4 @@
 # Hello World!
 Getting started with Hello World!
 First README comment
+Second README comment
```

Comment:
lol, I'm lacking imagination and alcohol at this time of night to do anything more interesting.